__tests__/endpoints.test.js

```diff
@@ -33,6 +35,14 @@ describe("/api/topics", () => {
         });
       });
   });
+  test.skip("GET 404 - Unable to fetch topics from the correct endpoint", () => {
+    return request(app)
```

Comment:
I would just remove these 404 tests. Appreciate that you are trying to be super thorough with your testing, and there should be a catch all 404 test for any request to a path that doesn't exist, but /api/topics and /api/articles are valid routes! Also as mentioned below, if a table has no data inside it can still be considered a valid request if we want (we could just do some handling on the frontend to feedback that there's no data available here currently, you won't be able to trigger that error currently unless you delete your test data which I wouldn't recommend/please don't do that 😆).